apis/apiextensions/v1/composition_webhooks.go

```diff
@@ -36,10 +36,13 @@ type CompositionValidationMode string
 
 var (
 	// DefaultCompositionValidationMode is the default validation mode for Compositions.
-	DefaultCompositionValidationMode = CompositionValidationModeLoose
+	DefaultCompositionValidationMode = CompositionValidationModeWarn
```

Comment:
Confused about the fact that we were running in Loose before, I was expecting to see a transition from `strict` to `warn` here instead 🤔

Reply:
Or should I read them as, `looseError`, `strictWarn`, `strictError` ?

Reply:
> what's confusing is the annotations being `crossplane.io/composition-validation-mode` instead of something more specific as `crossplane.io/composition-schema-aware-validation-mode`

Yes, I think that is exactly why I am confused and changing as above would make it more clear.

Reply:
have a look at my latest commit: https://github.com/crossplane/crossplane/pull/4892/commits/a93e3c0a485a7ee09eea9481116018fa7677684e